app/assets/javascripts/create.js

```diff
@@ -0,0 +1,24 @@
+$(document).ready(function() {
+  $('.join').click(function(event) {
+    event.preventDefault();
+    var user_id = $('.join').data().userId;
+    var opp_id = $('.join').data().opportunityId;
+    var request = $.ajax({
+      method: "POST",
+      url: "/api/v1/volunteer_lists",
+      data: {
+        volunteer_list: {
+          user_id: user_id,
+          opportunity_id: opp_id
```

Comment:
Identifier 'opportunity_id' is not in camel case.<br>Identifier 'opp_id' is not in camel case.